internal/driver/commands.go

```diff
@@ -273,7 +289,9 @@ func usage(commandLine bool) string {
 	}
 
 	help = help + strings.Join(commands, "\n") + "\n\n" +
-		"  Options:\n"
+		"  Options:\n" +
+		"    General format is <option>=<value>\n" +
+		"    <f> is a float, <n> is an integer, and <s> is a string\n\n"
```

Comment:
What about booleans? E.g. how to set a boolean to false?

Reply:
Done.